packages/rn-tester/js/components/RNTesterEmptyBookmarksState.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
Can you add the copyright header to this file?

Reply:
Done!